libphal/phal_exception.C

```diff
@@ -5,18 +5,34 @@ namespace openpower::phal
 {
 namespace exception
 {
+//taken from error_info_defs.H, including the header file breaks logging and
+//debug collector as they need to include ekb in include path
+enum errlSeverity_t
```

Comment:
As it is a new enum can we use `enum class` in place of only `enum` if that doesn't break anything anywhere?

Reply:
Also the same enum is declared and defined in the `phal_exception.H` file at LN12 then why do we need to redefine it here once again?

Reply:
this is taken from error_info_defs.H so want to keep the naming convention and e.t.c same.

I tried reusing enum from error_info_defs.H by including error_info_defs.H in phal_exceptions.H but as phal_exceptions.H is used in other repositories where they do not have include path for libkekb and ekb running into issues. Do not want to make openpower-debug-collector, ipl, logging depended on ekb for just this enum so hard coding here.